qiskit/algorithms/quantum_time_evolution/variational/var_qrte.py

```diff
@@ -57,9 +59,11 @@ def __init__(
             error_based_ode: If False use the provided variational principle to get the parameter
                                 updates.
                              If True use the argument that minimizes the error error_bounds.
+                             Deprecated if error is not being computed
```

Comment:
```suggestion
                             Deprecated if error is not being computed.
```
Deprecated? In what sense?